### DISCUSSION TITLE
Problem with audio output detection

### DISCUSSION THREAD

**Discussion by mapomme1108** [MusicPlayerDaemon/MPD#2111] [Q&A]
Hello,

I have a HTPC connected to an audio/video receiver with HDMI

Windows 11 is installed on the HTPC

I use Media Player Daemon but if I start it when the av receiver is not powered on, it does not start because the HDMI audio output to the av receiver is not detected by Windows so Music Player Daemon does not detect it either

I would like to know if it is possible to start Music Player Daemon even if the HDMI audio output (which is configured in mpd.conf) is not detected yet.

Thank you if you can help me

**Comment by MaxKellermann** [comment]
https://www.musicpd.org/help/

**Comment by geneticdrift** [comment]
Try adding a [`null`](https://mpd.readthedocs.io/en/latest/plugins.html#id34) audio output.
```
audio_output {
        type            "null"
        name            "Null Output"
        mixer_type      "none"                  # optional
}
```

**Comment by mapomme1108** [comment]
Hello,

Thank you for your answers.

My version of Music Player Daemon is 0.23.6

Here are my audio output in mpd.conf 

Analog output :
audio_output {
	type		"winmm"
	name		"Jack"
	device		"Haut-parleurs (2- High Definition Audio Device)"
**           or
**	device		"0"		# optional
	mixer_type	"software"
}

HDMI output :
audio_output {
	type		"winmm"
	name		"HDMI"
	device		"marantz-AVR (Son Intel(R) pour écrans)"
**		or
**	device		"0"		# optional
	mixer_type	"software"
}

The error message if I start Music Player Daemon with the av receiver off is :

`exception: Failed to configure output in line 318; device "marantz-AVR (Son Intel(R) pour Ã©crans)" is not found`

**Comment by mapomme1108** [comment]
Hello,

I have changed my audio output in mpd.conf

from

`device "marantz-AVR (Son Intel(R) pour écrans)"`

to

`device "0"`

and it works !!

I also want to configure the analog output.
Do you know how I can have the device number list of all the audio devices?
